svn/common.py

```diff
@@ -22,12 +22,32 @@ def __init__(self, message):
 
 
 class CommonClient(object):
+    """Subversion client
```

Comment:
This also encourages people to use `Common` directly (which I've seen before), which is not correct.